### PR TITLE
Clean up code

### PR DESCRIPTION
**Pull request by JacksonTian**

### CODE REVIEW

File: lib/WebSocket.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ util.inherits(WebSocket, EventEmitter);
  * Ready States
  */
 ["CONNECTING", "OPEN", "CLOSING", "CLOSED"].forEach(function each(state, index) {
-    WebSocket.prototype[state] = WebSocket[state] = index;
+    WebSocket[state] = index;
```
Comment (ericmdantas) [inline lib/WebSocket.js]: Wouldn't this break those who use the `wsInstance.CLOSE`, etc?

Comment (JacksonTian) [inline lib/WebSocket.js]: Maybe deprecation should be better?

Comment (ericmdantas) [inline lib/WebSocket.js]: Well, I wouldn't deprecate it if it was really needed for whatever reason. I guess there's a few people out there who test if a client is ready to go by doing:

```
if (client.readyState === client.OPEN) {
  // send data
} 
```

I currently use the above.

Comment (JacksonTian) [inline lib/WebSocket.js]: roll backed.

**Comment by 3rd-Eden** [comment]
Looks sane to me, thanks for the contribution.

**Pull request merged**